src/main/java/io/fabric8/elasticsearch/plugin/acl/DynamicACLFilter.java

```diff
@@ -78,407 +77,424 @@
-	private final UserProjectCache cache;
-	private final String proxyUserHeader;
-	private final Client esClient;
-	private final String searchGuardIndex;
-	private final String kibanaIndex;
-	private final String kibanaVersion;
-	private final String userProfilePrefix;
-	private final Settings settings;
-	private final ReentrantLock lock = new ReentrantLock();
-	private final Condition syncing = lock.newCondition();
-
-	private final String kbnVersionHeader;
-	private final String[] operationsProjects;
-
-	private Boolean enabled;
-	private Boolean seeded;
-
-	private final boolean use_cdm;
-	private final String cdm_project_prefix;
-
-	@Inject
-	public DynamicACLFilter(final UserProjectCache cache, final Settings settings, final Client client){
-		this.cache = cache;
-		this.logger = Loggers.getLogger(getClass(), settings);
-		this.proxyUserHeader = settings.get(SEARCHGUARD_AUTHENTICATION_PROXY_HEADER, DEFAULT_AUTH_PROXY_HEADER);
-		this.searchGuardIndex = settings.get(SEARCHGUARD_CONFIG_INDEX_NAME, DEFAULT_SECURITY_CONFIG_INDEX);
-		this.userProfilePrefix = settings.get(OPENSHIFT_ES_USER_PROFILE_PREFIX, DEFAULT_USER_PROFILE_PREFIX);
-		this.kibanaIndex = settings.get(KIBANA_CONFIG_INDEX_NAME, DEFAULT_USER_PROFILE_PREFIX);
-		this.kibanaVersion = settings.get(KIBANA_CONFIG_VERSION, DEFAULT_KIBANA_VERSION);
-		this.kbnVersionHeader = settings.get(KIBANA_VERSION_HEADER, DEFAULT_KIBANA_VERSION_HEADER);
-
-		this.operationsProjects = settings.getAsArray(OPENSHIFT_CONFIG_OPS_PROJECTS, DEFAULT_OPENSHIFT_OPS_PROJECTS);
-		this.use_cdm = settings.getAsBoolean(OPENSHIFT_CONFIG_USE_COMMON_DATA_MODEL, OPENSHIFT_DEFAULT_USE_COMMON_DATA_MODEL);
-		this.cdm_project_prefix = settings.get(OPENSHIFT_CONFIG_PROJECT_INDEX_PREFIX, OPENSHIFT_DEFAULT_PROJECT_INDEX_PREFIX);
-
-		logger.debug("searchGuardIndex: {}", this.searchGuardIndex);
-
-		this.settings = settings;
-
-		this.seeded = false;
-		this.enabled = settings.getAsBoolean(OPENSHIFT_DYNAMIC_ENABLED_FLAG, OPENSHIFT_DYNAMIC_ENABLED_DEFAULT);
-		// This is to not have SG print an error with us loading the SG_SSL plugin for our transport client
-		System.setProperty("sg.nowarn.client", "true");
-
-		/** Build the esClient as a transport client **/
-		String clusterName = settings.get("cluster.name");
-		String keystore = settings.get(SG_CLIENT_KS_PATH, DEFAULT_SG_CLIENT_KS_PATH);
-		String truststore = settings.get(SG_CLIENT_TS_PATH, DEFAULT_SG_CLIENT_TS_PATH);
-		String kspass = settings.get(SG_CLIENT_KS_PASS, DEFAULT_SG_CLIENT_KS_PASS);
-		String tspass = settings.get(SG_CLIENT_TS_PASS, DEFAULT_SG_CLIENT_TS_PASS);
-		String kstype = settings.get(SG_CLIENT_KS_TYPE, DEFAULT_SG_CLIENT_KS_TYPE);
-		String tstype = settings.get(SG_CLIENT_TS_TYPE, DEFAULT_SG_CLIENT_TS_TYPE);
-
-		Settings.Builder settingsBuilder = Settings
-                .builder()
-                .put("path.home", ".")
-                .put("path.conf", ".")
+public class DynamicACLFilter extends RestFilter implements ConfigurationSettings {
```

Comment:
This looks like almost a complete reformatting of the spacing in the file.  Is this right?